tests/e2e/utils/versions_test.go

```diff
@@ -336,13 +406,17 @@ func TestGetVersionMapping(t *testing.T) {
 	}
 
 	for i, tc := range testContexts {
+		// run the function, but use the testMapper,
```

Comment:
Can you add a `name` field to each test and call the `run` function so we can execute these as multiple tests? That we way we will know which one failed if one ever does. Like this:
![image](https://user-images.githubusercontent.com/3431073/205987660-8d67022f-854e-4c60-919f-f470a03f7138.png)